src/core/favorite/favorite.service.ts

```diff
@@ -46,12 +49,70 @@ export class FavoriteService {
       },
     }
   ];
+  favoritesCitiesNames: string[] = ['Toronto','Oslo','Odessa']; //names
 
-  constructor(public favoriteStateService: FavoriteStateService) {
+  constructor(public favoriteStateService: FavoriteStateService, private weatherService: WeatherService) {
     this.favorites = this.testFavorites;
+    // this.getFavorites().subscribe((v)=>{console.log(v)});
+
+    this.getFavorite().subscribe(value=> {
+      console.log(value);
+    });
+  }
+
+
+
+  getFavorite(): Observable<CityWeatherInfo>{
```

Comment:
Communicate with Kate and Oleksii, this is old implementation or I will wait their implementation